src/color_chooser.rs

```diff
@@ -55,9 +50,11 @@ impl<O: IsA<ColorChooser>> ColorChooserExt for O {
         }
     }
 
-    fn add_palette(&self, orientation: Orientation, colors_per_line: i32, colors: Vec<GdkRGBA>) {
+    fn add_palette(&self, orientation: Orientation, colors_per_line: i32, colors: Vec<RGBA>) {
```

Comment:
I don't quite follow, are you saying that the `colors` argument should **not** be `Vec<RGBA>` or are you saying my conversion from `Vec<RGBA>` to `*mut gdk_ffi::GdkRGBA` is wrong?